benchmark/alpa/run_exp.py

```diff
@@ -33,17 +35,20 @@ def run_exp(cluster_settings, suite_name, benchmark_settings=None):
     "gpt": ("gpt.grid_search_auto", {}),
     "moe": ("moe.grid_search_auto", {}),
     "wresnet": ("wresnet.grid_search_auto", {}),
-    "gpt_inference": ("gpt_inference.profile", {}),
+    "gpt_inference": ("gpt_inference.profile", {
+        "niter": 10,
+        "profile_stage_execution_time": True
+    }),
     "gpt_no_embedding_inference": ("gpt_no_embedding_inference.profile", {}),
     "gpt_inference_streaming": ("gpt_inference.profile", {
         "profile_driver_time": True
     }),
 }
-cluster_settings = [(8, 8), (4, 8), (2, 8), (1, 8), (1, 4), (1, 2), (1, 1)]
-
+cluster_settings = [(4, 8), (3, 8), (2, 8), (1, 8), (1, 4), (1, 2), (1, 1)]
```

Comment:
Do not delete (8,8). There are other experiments using (8,8). 
I think (8,8) will be safely skipped for your case so adding it is okay.